src/javascript/app_2/App/Components/Elements/ContractAudit/contract-audit.jsx

```diff
@@ -0,0 +1,110 @@
+import moment                  from 'moment';
+import classNames              from 'classnames';
+import PropTypes               from 'prop-types';
+import React, { Component }    from 'react';
+import { Icon, IconArrowBold } from 'Assets/Common';
+import Localize                from '../localize.jsx';
+
+const Pair = ({ value, label }) => (
+    <div className='pair'>
+        <div className='pair__label'>
+            <Localize str={label} />
+        </div>
+        <div className='pair__value'>
+            {value}
+        </div>
+    </div>
+);
+class ContractAudit extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            is_visible: false,
+        };
+    }
+
+    toggleVisibility () {
+        this.setState({
+            is_visible: !this.state.is_visible,
+        });
+    }
+
+    getDuration () {
+        const {
+            tick_count,
+            date_start,
+            date_expiry,
+        } = this.props.contract;
+
+        const is_tick_contract = !!this.props.contract.tick_count;
```

Comment:
We already use the shorthand notation on line 34, we should just use `!!tick_count` instead of `!!this.props.contract.tick_count`

Reply:
Nice catch @ahkuma